apps/src/Observer.js

```diff
@@ -0,0 +1,52 @@
+/** @file Helper for consuming ObservableEvents. */
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+ eqeqeq: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
```

Comment:
Same comment about jshint rules. I suspect you're just trying to move to a world where we're more strict? I'm fine with that, but do think it deserves a comment explaining.